README.md

```diff
@@ -145,6 +145,13 @@ git clone https://github.com/ggerganov/llama.cpp
 cd llama.cpp
 make
 
+#For Windows and CMake, use the following command instead:
```

Comment:
the instruction looks good. can we also mentioned that the user needs to install visual studio?